contrib/aws/create

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+set -eo pipefail -o nounset
+
+export DEIS_NUM_INSTANCES=3
+
+function aws-setup-keypair {
+  local deis_auth_key="${1}"
+
+  rigger-log "Importing ${deis_auth_key} keypair to EC2"
+
+  # TODO: don't hardcode --key-names
+  if ! aws ec2 describe-key-pairs --key-names "deis" >/dev/null ; then
+    rigger-log "Importing ${deis_auth_key} keypair to EC2"
+    aws ec2 import-key-pair --key-name deis \
+        --public-key-material file://${deis_auth_key}.pub \
+        --output text
+  fi
+}
+
+function aws-provision-cluster {
+  local stack_name="${1}"
+
+  # customize cloudformation.json to use m3.medium instances
```

Comment:
1) This comment makes no sense with regards to what actually happens in the code
2) How are we supposed to customize instance type and other cloudformation parameters when this overwrites the cloudformation.json?